string[] condition_names
string[] pre_action_consequence_names
string counter_factual_action_name

---

bool success
string[] post_action_consequence_names
